Directory.Build.props

```diff
@@ -12,6 +12,7 @@
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <RepositoryUrl>https://github.com/enigmatry/entry-building-blocks</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
+    <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
```

Comment:
Out of curiosity what is this for?

Reply:
Yep. I also use these files to create a hash, which is used as the cache key for the nuget cache.

Reply:
Interesting that package.json was not enough, does this have any negative impact on something (performance, maintenance effort). Does this mean that we need to change all our projects to enable this flag so that azure devops build works?

Reply:
Yes, it does.